v2/passwordless/updating-supertokens.mdx

```diff
@@ -0,0 +1,54 @@
+---
+id: updating-supertokens
+title: Updating Supertokens SDKs and Core
+hide_title: true
+---
+
+<!-- COPY DOCS -->
+<!-- ./community/updating-supertokens.mdx -->
+
+# Updating Supertokens SDKs and Core
+
+Not all older releases of every supertokens Core, front-end SDK, and back-end SDK are compatible with each other. In order to maintain compatability
+and keep up on new features, we recommend updating and upgrading all of your SDKs regularly, starting with the supertokens core, then the back-end SDK, then the front-end SDK.
+
+The following is a link to each core and SDK changelog, containing steps to migrate from each version. You can use our [compatability table](../../community/compatibility-table) to see which versions you need to upgrade to.
+
+## Core Migration
+
+- ### Self-hosted Core
+[supertokens-core](https://github.com/supertokens/supertokens-core/blob/master/CHANGELOG.md)
+
+- ### Managed Core
+If you're using the managed core service, please email us at team@supertokens.com from your registered email to request a core update.
```

Comment:
This needs more clarity.. talk to @sattvikc here about how to actually run the migrations - suppose that users want to migrate from v5 to v9, do the users need to stop the core, run all the migrations from v5...v9 that are mentioned in the changelog, and then start the v9 core? Or stop v5, do migration of next version, start core of next version, then do migration of version after that, then start core for that version.. all the way till v9?

Reply:
Talked with him and updated the copy